utils.py

```diff
@@ -257,7 +253,7 @@ def service_get(company):
                     for service in v:
                         for i_k, i_v in service.items():
                             if (i_k == 'name'):
-                                if (i_v == 'service-' + company + '-' + counter + temp):
+                                if (i_v == 'service-' + company):
                                     print('Here isnide service')
```

Comment:
(minor: typo isnide/inside)

Reply:
@tisnik, Fixed it in this https://github.com/fabric8-analytics/f8a-3scale-connect-api/pull/10/commits/5694df15d25b5cc0d90a69c423c91f79ed5f5411